### PR TITLE
Initial content to the microsite

### LINKED ISSUE #28: Helios documentation
We need to add documentation about helios and how to use it in order to create the proper microsite.

Must to include:

 - [ ] What's Helios and how to import it
 - [ ] How to use Helios
 - [ ] Benchmarks
 - [ ] References and samples

This doc is apart from the Readme issue (#9)

### PR DESCRIPTION
**Pull request by AdrianRaFo**
Fixes #28

### CODE REVIEW

**Comment by 47degdev** [comment]
***Hood benchmark comparison:***
:red_circle: `Decoding` (Threshold: 108.423407)

Benchmark | Value
|--|--|
master_benchmark | 11585.823369
helios_benchmark | 10360.356776385775


:red_circle: `Parsing` (Threshold: 760.098092)

Benchmark | Value
|--|--|
master_benchmark | 23084.587761
helios_benchmark | 21251.040415542804

File: README.md [hunk 1]
```diff
@@ -22,45 +29,30 @@ dependencies {
...
-moshi  | 19017.079668 | 1202.955223 | ops/s |
-     
-## Running Benchmarks
+```bash
+./gradlew :helios-benchmarks: executeHeliosBenchmark
+```
+
+## Running Microsite
+
+To run locally the **Helios** microsite you need to execute first this:
```
Comment (nomisRev) [inline README.md]:
```suggestion
To run the **Helios** microsite locally, you need to execute the following command:
```

File: helios-docs/docs/docs/coding/README.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+```kotlin:ank:silent
+val jObject = JsObject(
+"name" to JsString("Elia"),
+"age" to JsNumber(23)
+)
+```
+
+## Custom Encoders
+
+To create a custom Encoder just just need to inherit from the Encoder interface and implement the `encode` method.
```
Comment (nomisRev) [inline helios-docs/docs/docs/coding/README.md]:
```suggestion
To create a custom Encoder, you need to inherit from the `Encoder` interface and implement the `encode` method.
```

File: helios-docs/docs/docs/coding/README.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+      Person(tuple.a, tuple.b)
+    }.fix()
+}
+
+personCustomDecoder.decode(personCustomJson)
+```
+
+## Navigation through Json
+
+You can also navigate through the Json using optics.
```
Comment (nomisRev) [inline helios-docs/docs/docs/coding/README.md]:
```suggestion
You can navigate `Json` using the `Json.path` DSL to select keys or traverse collections.
```

File: helios-docs/docs/docs/quickstart/README.md [hunk 4]
```diff
@@ -1,12 +1,17 @@
+---
+layout: docs
+title: QuickStart
+permalink: /docs/quickstart/
+---
+
 # QuickStart
 
-Once added helios to you project, let's start to add the needed imports:
+Once added helios to your project, let's start to add the needed imports:
```
Comment (nomisRev) [inline helios-docs/docs/docs/quickstart/README.md]:
```suggestion
Once Helios is added to your project, let's start to add the needed imports:
```

**Comment by AdrianRaFo** [comment]
@MaureenElsberry I would like to have a lang review here please

**Pull request merged**